crawlerr/config.py

```
@@ -88,17 +95,19 @@ def cors_origins_list(self) -> List[str]:
             log_path = Path(v)
             log_path.parent.mkdir(parents=True, exist_ok=True)
         return v
     
-    class Config:
-        env_file = ".env"
-        env_file_encoding = "utf-8"
-        case_sensitive = False
+    model_config = ConfigDict(
+        env_file=".env",
+        env_file_encoding="utf-8",
+        case_sensitive=False
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)